test/graphql/execution/executor_test.exs

```diff
@@ -37,6 +37,21 @@ defmodule GraphQL.Execution.Executor.ExecutorTest do
     assert_execute {~S[{ greeting(name: "Elixir") }], TestSchema.schema}, %{greeting: "Hello, Elixir!"}
   end
 
+  test "anonymous fragments are processed" do
+    schema = %GraphQL.Schema{
+      query: %GraphQL.ObjectType{
+        name: "X",
+        fields: %{
+          id:   %{type: "String", resolve: fn(p, _, _) -> p.id   end},
+          name: %{type: "String", resolve: fn(p, _, _) -> p.name end}
+        }
+      }
+    }
+    {:ok, doc} = Parser.parse  ~s| {id, ... { name } } |
+    data = %{id: 1, name: "Mark"}
+    assert Executor.execute(schema, doc, data) == {:ok, %{id: 1, name: "Mark"}}
```

Comment:
Think you can safely use the `assert_execute` helper here, (also query should only be a sigil when needed)